pkg/proxy/nftables/proxier.go

```diff
@@ -399,6 +401,19 @@ func (proxier *Proxier) setupNFTables(tx *knftables.Transaction) {
 		Comment: ptr.To("rules for kube-proxy"),
 	})
 
+	// Do an extra "add+delete" once to ensure all previous base chains in the table
+	// will be recreated. Otherwise, altering properties (e.g. priority) of these
+	// chains would fail the transaction.
+	proxier.flushOnce.Do(func() {
```

Comment:
You don't need a `sync.Once` here; a boolean is fine, since it's all being done under a mutex anyway.

Reply:
done